### DISCUSSION TITLE
PathFigure IsClosed

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#16670] [Q&A]
## Avalonia:

https://github.com/AvaloniaUI/Avalonia/blob/ec655a9f7e8cb53536890112807ddf5d1caae9da/src/Avalonia.Base/Media/PathFigure.cs#L14-L15

## WPF:

https://github.com/dotnet/wpf/blob/e7ddcf41e1342d889aa7273b836909bdf15154e2/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Generated/PathFigure.cs#L315-L323

As you can see while Avalonia defaults to True, WPF defaults to false. This can cause problems when porting from WPF to Avalonia.

Changing Avalonia  IsClosed default value would be a major breaking change. What could be the right solution?

**Comment by maxkatz6** [comment]
I don't feel like we can make this behavioral change at this point.
Aside from WPF portability, this change wouldn't bring any improvements either.

**Reply by workgroupengineering** [reply]
What do you think if we add a disableable warning?
